app/src/androidTest/java/org/simple/clinic/TestData.kt

```diff
@@ -483,6 +485,29 @@ class TestData @Inject constructor(
         recordedAt = recordedAt)
   }
 
+  fun bloodSugarPayload(
+      uuid: UUID = UUID.randomUUID(),
+      bloodSugarType: BloodSugarMeasurementType = BloodSugarMeasurementType.random(),
+      bloodSugarValue: Int = faker.number.between(0, 300),
+      patientUuid: UUID = UUID.randomUUID(),
+      facilityUuid: UUID = qaUserFacilityUuid(),
+      createdAt: Instant = Instant.now(),
+      updatedAt: Instant = Instant.now(),
+      deletedAt: Instant? = null,
+      recordedAt: Instant = Instant.now()
+  ) = BloodSugarMeasurementPayload(
+      uuid = uuid,
+      bloodSugarType = bloodSugarType,
+      bloodSugarValue = bloodSugarValue,
+      patientUuid = patientUuid,
+      facilityUuid = facilityUuid,
+      userUuid = qaUserUuid(),
```

Comment:
Could you make this a method parameter like all the others, please?

Reply:
done.